### ISSUE TITLE
eclipse content assist not activating

### ISSUE THREAD

**Issue by jagretz** [palantir/eclipse-typescript#256]
Hello,

I installed typescript (Windows 7), running cmd "tsc -v" shows "message TS6029: Version 1.5.0-beta". I installed the plugin via the instructions, I checked the node path and checked "Enable auto activation" in the project settings and the eclipse preferences but cannot seem to get the content assist to work at all.

My eclipse version I am using is STS Spring Tool Suite 3.6.4.RELEASE

Any help or suggestions would be appreciated.

**Comment by derekcicerone** [comment]
You may have to enable the TypeScript builder for each project to make auto-complete work.

**Comment by jagretz** [comment]
I knew there was something else I forgot to mention; Yes, the TypeScript Builder is enabled.

**Comment by derekcicerone** [comment]
Ok, the next thing to check is the error log view to see if something is breaking. Also, how are you trying to activate auto-completion?

**Comment by jagretz** [comment]
I don't see anything on the console, the problems view or the error log view.
I am trying to auto-complete using the "." operator and then waiting. Then I try hitting crtl + space which should bring up code completion options available to me, but does not.

**Comment by derekcicerone** [comment]
Hmm weird, sorry, not sure I have more suggestions to try out in that case. From what you are describing it should work. The file name ends with .ts right?

**Comment by derekcicerone** [comment]
Also, the file needs to open in the TypeScript editor view, not a normal text editor view...

**Comment by jagretz** [comment]
Well this is a little embarrassing... I set the file associations and added the Typescript editor as the default, but it was still choosing the javascript editor (I didn't realize that). I removed the js editor and now the content assist works... heh
Thanks for mentioning that little pointer. Sometimes the most annoying problems have the easiest solutions, like plugging in your computer before trying turn it on.
